### PR TITLE
Homepage Plugins nudge - UI

### PR DESCRIPTION
**Pull request by aratidgr8**
## Proposed changes

JIRA: https://jira.newfold.com/browse/PRESS0-2212

**Given:**
BlueHost home page with WordpPress solutions package
**When:**
Customers have purchase Creator/Service/Ecommerce solution package
**Then:**
Display "Explore your Plugins and Tools" section on brand plugin home page
[Figma](https://www.figma.com/design/b6qZ8lCZQc28494PiVnCJv/PRESS0-1672)

**CREATOR:**
content creator

**SERVICES:**
services

**ECOMMERCE:**
ecomm

**NONE:**
none

JIRA: https://jira.newfold.com/browse/PRESS0-2229

**Given:**
BlueHost home page,
**When:**
Customers have not purchased any WordPress solutions (Creator/Service/Ecommerce) package
**Then:**
Show "Upgrade your plan for success" section

**No existing plan**
no existing plan

JIRA: https://jira.newfold.com/browse/PRESS0-2231

**Given:**
BlueHost home page with WordpPress solutions package
**When:**
Customers have purchase Creator/Service/Ecommerce solution package
**Then:**
Store page should have "View all your plugins & tools" CTA

Overview v2 - ecomm users - single use

Overview v2 - ecomm users

JIRA: https://jira.newfold.com/browse/PRESS0-1719

**Given:**
A customer has either the eCommerce, Creator, or Service solution addon
**When:**
They navigate to their Home page of the branded plugin 
**Then:**
Next Steps includes a step to "Explore the features included with your solution"

Screenshot 2024-09-26 at 3 17 32 PM

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/bluehost/.github/blob/master/.github/contributing.md) doc
- [ ] I have viewed my change in a web-browser
- [ ] Linting and tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by wpscholar** [comment]
@aratidgr8 The solutions/entitlements API is now available on Hiive prod. Can you go ahead and integrate with the live API please? 

The endpoint `/api/sites/v1/entitlements` and will return a `solutions` endpoint with the data you need.

**Comment by aratidgr8** [comment]
> @aratidgr8 The solutions/entitlements API is now available on Hiive prod. Can you go ahead and integrate with the live API please?
> 
> The endpoint `/api/sites/v1/entitlements` and will return a `solutions` endpoint with the data you need.

Tried using rest route as mentioned in [PR](https://github.com/newfold-labs/wp-module-solutions/pull/2/files) getting attached error.
Screenshot 2024-10-03 at 5 28 46 PM
Screenshot 2024-10-03 at 5 28 59 PM

@girish-lokapure  please confirm if I need to pass any parameter to run this is on local environment. Thanks.

**Comment by aratidgr8** [comment]
> @aratidgr8 The solutions/entitlements API is now available on Hiive prod. Can you go ahead and integrate with the live API please?
> 
> The endpoint `/api/sites/v1/entitlements` and will return a `solutions` endpoint with the data you need.

@wpscholar Code changes done as per `entitlement` API. Please help me with PR review. Thanks!

File: src/configs/OnboardingList.config.js [hunk 1]
```diff
@@ -79,6 +84,23 @@ export function OnboardingListDefinition(props) {
...
+      {
+        name:"Explore the features included with your solution",
+        id: "mypluginsandtools",
+        text: __("Explore the features included with your solution", "wp-module-ecommerce"),
+        state: {
+          isAvailable: (queries) => queries?.settings?.hasBrandSignedUp,
+          isCompleted: (queries) => !(queries?.settings?.hasBrandSignedUp),
+          url: () =>
+            `${window.location.href.split('#')[0]}#/my_plugins_and_tools`,
+          target: () => "_blank",
```
Comment (ramyakrishnai) [inline src/configs/OnboardingList.config.js:96]: Hi @aratidgr8 / @searunt , can you please confirm , in the next steps when user clicks on `Explore the features included with your solution` are we expecting it to open it in new tab ??

Comment (aratidgr8) [reply src/configs/OnboardingList.config.js]: It will open in same tab as discussed with @wpscholar

**Review by diwanshuster — APPROVED** [review]

**Review by ramyakrishnai — APPROVED** [review]

**Review by circlecube — APPROVED** [review]

**Pull request merged**